docs/source/usersguide/input.rst

```diff
@@ -714,6 +709,47 @@ survival biasing, otherwise known as implicit capture or absorption.
 
   *Default*: false
 
+.. _temperature_default:
+
+``<temperature_default>`` Element
+---------------------------------
+
+The ``<temperature_default>`` element specifies a default temperature in Kelvin
+that is to be applied to cells in the absence of an explicit cell temperature or
+a material default temperature.
+
+  *Default*: 293.6 K
+
+.. _temperature_method:
+
+``<temperature_method>`` Element
+--------------------------------
+
+The ``<temperature_method>`` element has an accepted value of "nearest" or
+"interpolation". A value of "nearest" indicates that for each cell, the nearest
+temperature at which cross sections are given is to be applied, within a given
+tolerance (see :ref:`temperature_tolerance`). A value of "interpolation"
+indicates that cross sections are to be interpolated between temperatures at
```

Comment:
I'd prefer we remove "interpolation" from the docs until we have it implemented.